server/src/app.ts

```diff
@@ -68,6 +68,10 @@ app.use((err, req, res, next) => {
     return next(err);
   }
 
+  if (req.route.path === '/api/surveyResults') {
```

Comment:
I don't feel like it is necessary to handle errors for specific endpoints differently than others. For example, we don't do anything special for the `/api/token` and `/api/config` endpoints, so I am worried we are adding this unnecessarily and establishing a pattern that I think is not very scalable for future endpoints we might add.

I am completely fine without this part, and let Contoso deal with errors if they don't like the default 500 status code.

What are your thoughts on this? @hansung-msft @rvenugopal-msft

Reply:
Yea as we discussed, I am okay with generic error. If we all agree with generic error, I will revert the change. 

cc @rvenugopal-msft

Reply:
Thats a fair point. I think that makes sense to me . If we want to do that we handle it as a generic controller logic not just for one endpoint.

Reply:
Sounds good! I will revert the changes and push it again. Thanks for sharing the opinions guys.